schemas/bolt-inventory.schema.json

```diff
@@ -22,6 +22,10 @@
     },
     "vars": {
       "$ref": "#/definitions/vars"
+    },
+    "version": {
+      "description": "The version of the inventory file.",
```

Comment:
This change shouldn't be in this commit. Nor should the one in `bolt-config.schema.json`

Reply:
Whoops. Updated.